guides/source/install_ruby_on_rails.md

```diff
@@ -0,0 +1,131 @@
+* [Ubuntu](#install-ruby-on-ubuntu)
+* [Windows](#install-ruby-on-windows)
+
+TIP: Any commands prefaced with a dollar sign `$` should be run in the terminal.
+
+### Install Ruby on macOS
+
+You'll need macOS Catalina 10.15 or newer to follow these instructions.
+
+For macOS, you'll need Xcode Command Line Tools and Homebrew to install dependencies needed to compile Ruby.
+
+Open Terminal and run the following commands:
+
+```shell
+# Install Xcode Command Line Tools
+$ xcode-select --install
+
+# Install Homebrew and dependencies
+$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+$ echo 'export PATH="/opt/homebrew/bin:$PATH"' >> ~/.zshrc
+$ source ~/.zshrc
+$ brew install openssl@3 libyaml gmp rust
+
+# Install Mise version manager
+$ curl https://mise.run | sh
+$ echo 'eval "$(~/.local/bin/mise activate)"' >> ~/.zshrc
+$ source ~/.zshrc
+
+# Install Ruby globally with Mise
+$ mise use -g ruby@3
+```
+
+### Install Ruby on Ubuntu
+
+You'll need Ubuntu Jammy 22.04 or newer to follow these instructions.
+
+Open Terminal and run the following commands:
+
+```bash
+# Install dependencies with apt
+$ sudo apt update
+$ sudo apt install build-essential rustc libssl-dev libyaml-dev zlib1g-dev libgmp-dev
+
+# Install Mise version manager
+$ curl https://mise.run | sh
+$ echo 'eval "$(~/.local/bin/mise activate)"' >> ~/.bashrc
+$ source ~/.bashrc
+
+# Install Ruby globally with Mise
+$ mise use -g ruby@3
+```
+
+### Install Ruby on Windows
+
+The Windows Subsystem for Linux will provide the best experience for Ruby on Rails development on Windows. It runs Ubuntu inside of Windows which allows you to work in an environment that is close to what your servers will run in production.
+
+You will need Windows 11 or Windows 10 version 2004 and higher (Build 19041 and higher).
+
+Open PowerShell or Windows Command Prompt and run:
+
+```bash
+$ wsl --install --distribution Ubuntu-24.04
+```
+
+You may need to reboot during the installation process.
+
+Once installed, you can open Ubuntu from the Start menu. Enter a username and password for your Ubuntu user when prompted.
+
+Then run the following commands:
+
+```bash
+# Install dependencies with apt
+$ sudo apt update
+$ sudo apt install build-essential rustc libssl-dev libyaml-dev zlib1g-dev libgmp-dev
+
+# Install Mise version manager
+$ curl https://mise.run | sh
+$ echo 'eval "$(~/.local/bin/mise activate bash)"' >> ~/.bashrc
+$ source ~/.bashrc
+
+# Install Ruby globally with Mise
+$ mise use -g ruby@3
+```
+
+Verifying your Ruby install
+---------------------------
+
+Once Ruby is installed, you can verify it works by running:
+
+```bash
+$ ruby --version
+ruby 3.3.6
+```
+
+Installing Rails
+------------------------
+
+A "gem" in Ruby is a self-contained package of a library or Ruby program. We can use Ruby's `gem` command to install the latest version of Rails and its dependencies from [RubyGems.org](https://rubygems.org).
```

Comment:
```suggestion
A "gem" in Ruby is a self-contained package of a library or Ruby program. We can use Bundler's `gem` command to install the latest version of Rails and its dependencies from [RubyGems.org](https://rubygems.org).
```
(although the line is blurred since Bundler is built-in)

Reply:
We'd probably need to explain Bundler here too?

Reply:
Yeah, it might be too much for an intro.